apps/settings/js/panels/browsing_privacy/panel.js

```diff
@@ -3,18 +3,12 @@ define(function(require) {
 
   var SettingsPanel = require('modules/settings_panel');
   var BrowsingPrivacy = require('panels/browsing_privacy/browsing_privacy');
+  var DialogService = require('modules/dialog_service');
```

Comment:
remember to exclude `modules/dialog_service` in `js/config/require.js` when you use it